crates/topos-api/src/graphql/certificate.rs

```diff
@@ -0,0 +1,49 @@
+use async_graphql::SimpleObject;
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, Default, Serialize, Deserialize, SimpleObject)]
+pub struct SubnetId {
```

Comment:
Should have been in a `subnet` file not in `certificate`

Reply:
Yeah, the structuring of the folder was a bit rushed. Happy to move things around. I was hoping we get another endpoint or two in soon, so we can figure things out then ones we have more sample data.